app/backend/wells/views_v2.py

```diff
@@ -487,3 +488,33 @@ def list(self, request, *args, **kwargs):
         response['Content-Disposition'] = 'attachment; filename="search-results.{ext}"'.format(ext=renderer.format)
 
         return response
+
+
+class WellAquifer(ListAPIView):
+    """ returns aquifer info for a range of wells """
+
+    model = Well
+    serializer_class = WellAquiferSerializerV2
+    filter_backends = (GeometryFilterBackend,)
+    swagger_schema = None
+
+    def get_queryset(self):
+        qs = Well.objects.all()
+
+        if not self.request.user.groups.filter(name=WELLS_EDIT_ROLE).exists():
+            qs = qs.exclude(well_publication_status='Unpublished')
+
+        # allow comma separated list of wells by well tag number
+        wells = self.request.query_params.get('wells', None)
+        if wells:
+            wells = wells.split(',')
+
+            for w in wells:
+                if not w.isnumeric():
+                    raise ValidationError(detail='Invalid well')
+
+            wells = map(int, wells)
+
+            qs = qs.filter(well_tag_number__in=wells)
+
+        return qs
```

Comment:
I believe the serializer will make a SQL call for each aquifer in the returned results. Add a `.select_related` so all the aquifer data is fetched once?